pkger/service.go

```diff
@@ -1230,27 +1320,46 @@ func (s *Service) apply(ctx context.Context, coordinator *rollbackCoordinator, o
 
 	// secondary resources
 	// this last grouping relies on the above 2 steps having completely successfully
-	secondary := []applier{s.applyLabelMappings(pkg.labelMappings())}
+	secondary := []applier{
+		s.applyLabelMappings(pkg.labelMappings()),
+		s.applyLabelMappingsV2(state.labelMappings),
```

Comment:
original implementation will eventually die in a fire once new model gets adopted